impeller/renderer/backend/gles/device_buffer_gles.h

```diff
@@ -33,8 +39,9 @@ class DeviceBufferGLES final
   ReactorGLES::Ref reactor_;
   HandleGLES handle_;
   std::string label_;
-  mutable std::shared_ptr<fml::Mapping> data_;
-  mutable bool uploaded_ = false;
+  mutable std::shared_ptr<Allocation> buffer_;
```

Comment:
I think a better name for this would be `backing_store_`.

Reply:
Done.